src/main/java/frc/trigon/robot/subsystems/roller/Roller.java

```diff
@@ -0,0 +1,52 @@
+package frc.trigon.robot.subsystems.roller;
+
+import frc.trigon.robot.subsystems.MotorSubsystem;
+import org.littletonrobotics.junction.Logger;
+
+public class Roller extends MotorSubsystem {
+    private final static Roller INSTANCE = new Roller();
+    private final RollerIO rollerIO = RollerIO.generateIO();
+    private final RollerInputsAutoLogged rollerInputs = new RollerInputsAutoLogged();
+    private RollerConstants.RollerState targetState = RollerConstants.RollerState.STOPPED;
+
+    public static Roller getInstance() {
+        return INSTANCE;
+    }
+
+    private Roller() {
+    }
+
+    @Override
+    public void stop() {
+        rollerIO.stopMotor();
+        rollerIO.setTargetVelocity(0);
+    }
```

Comment:
You should update the mechanism here, not the target velocity (it is already set to 0 when using stopMotor